### ISSUE TITLE
i got it error

### ISSUE THREAD

**Issue by hackersden09** [RoliSoft/ReconScan#11]
root@kali:~/Desktop/weapons/ReconScan# ./vulnscan.py 'Apache 2.2.8'
Traceback (most recent call last):
  File "./vulnscan.py", line 24, in <module>
    from libnmap.parser import NmapParser
ModuleNotFoundError: No module named 'libnmap'

**Comment by Gr1mmie** [comment]
yeah I had the same issue, tried installing the module but still nothing

**Comment by Jpmij** [comment]
pip install python-libnmap
